### PR TITLE
Update for 1.0 release

### PR DESCRIPTION
**Pull request by dgryski**

### CODE REVIEW

**Comment by dgryski** [comment]
I should probably add 0.29 to the testing matrix too.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # compute-sdk-go
 
-Experimental Go SDK for building [Compute@Edge](https://www.fastly.com/products/edge-compute/serverless) applications with [Go](https://go.dev) (1.21+) and [TinyGo](https://tinygo.org/) (0.28.1+).
```
Comment (awilliams-fastly) [inline README.md:3]: `https://www.fastly.com/products/edge-compute/serverless` is a 301 redirect to `https://www.fastly.com/products/edge-compute`. May be worth updating.

**Review by joeshaw — APPROVED** [review]
👍

**Review by awilliams-fastly — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dgryski) [reply README.md:3]: https://github.com/fastly/compute-sdk-go/pull/79/commits/36d8fed2dcbd45772aaa8b63c9e196ecf9fb5cc9

**Pull request merged**